File: DoWhile.java
import java.util.Scanner;

public class DoWhile {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        char decision;

        do {
            System.out.print("Enter the first number: ");
            int num1 = scanner.nextInt();

            System.out.print("Enter the second number: ");
            int num2 = scanner.nextInt();

            int sum = num1 + num2;
            System.out.println("The sum is: " + sum);

            System.out.print("Do you want to perform the operation again? (Y/N): ");
            decision = scanner.next().charAt(0);
        } while (decision == 'Y' || decision == 'y');

    }
}

File: DoWhile.py
while True:
    num1 = float(input("Enter the first number: "))
    num2 = float(input("Enter the second number: "))

    sum = num1 + num2
    print("The sum is:", sum)

    choice = input("Do you want to perform the operation again? (yes/no): ")
    if choice.lower() != 'yes':
        break